cmd/kubeadm/app/kubeadm.go

```diff
@@ -56,7 +54,6 @@ func getEnvParams() map[string]string {
 }
 
 func Run() error {
-	CommandLine = pflag.NewFlagSet(os.Args[0], pflag.ContinueOnError)
```

Comment:
You are right, this actually didn't do anything. I've added it when I was trying to get rid of the flags which leaked from some other pkgs, but now hide those explicitly and it works well.